airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/bulk/query.py

```diff
@@ -988,17 +990,60 @@ class InventoryLevel(ShopifyBulkQuery):
         "new_record": "InventoryLevel",
     }
 
+    # quantity related fields and filtering options
+    quantities_names_filter: List[str] = [
+        '"available"',
+        '"incoming"',
+        '"committed"',
+        '"damaged"',
+        '"on_hand"',
+        '"quality_control"',
+        '"reserved"',
+        '"safety_stock"',
+    ]
+    # quantities fields
+    quantities_fields: List[str] = [
+        "id",
+        "name",
+        "quantity",
+        "updatedAt",
+    ]
+
     inventory_levels_fields: List[Field] = [
         "__typename",
         "id",
-        Field(name="available"),
         Field(name="item", fields=[Field(name="id", alias="inventory_item_id")]),
         Field(name="updatedAt"),
     ]
 
+    def _quantities_query(self) -> Query:
+        """
+        Defines the `quantities` nested query.
+        """
+
+        return Query(
+            name="quantities",
+            arguments=[Argument(name="names", value=self.quantities_names_filter)],
+            fields=self.quantities_fields,
+        )
+
+    def _process_quantities(self, quantities: Iterable[MutableMapping[str, Any]] = None) -> Iterable[Mapping[str, Any]]:
+        if quantities:
+            for quantity in quantities:
+                # save the original string id
```

Comment:
nit: I think the code is clear enough so that we don't need those comments

Reply:
I prefer to leave them there for the future clarity.